### DISCUSSION TITLE
renovate-ce-ee admin api?

### DISCUSSION THREAD

**Discussion by krohrsb** [mend/renovate-ce-ee#322] [Q&A] [answered]
Listed in the new overview for renovate-ce-ee https://github.com/mend/renovate-ce-ee/blob/main/docs/overview.md#overview it mentions a new set of administrative APIs. Is there any documentation around these? I could not locate. Thank you.

**Comment by rarkins** [comment, marked as answer]
I have created a PR here: https://github.com/mend/renovate-ce-ee/pull/332/files

I didn't write this API so waiting on review from some colleagues who were involved
